### PR TITLE
fix: rollback bump version on dev

### PR DESCRIPTION
**Pull request by IvanJobs**
Currently, versions on each branch are as below:
main:
- @microsoft/teamsfx-api: 0.2.0-rc.1
- @microsoft/teamsfx-cli: 0.2.0-rc.1
- @microsoft/teamsfx-core: 0.2.0-rc.1
- @microsoft/teamsfx: 0.2.0-rc.1
- ms-teams-vscode-extension: 2.1.0-rc.1

dev:
- @microsoft/teamsfx-api: 0.4.0-alpha.2 rollback to - @microsoft/teamsfx-api: 0.3.0-alpha.0
- @microsoft/teamsfx-cli: 0.4.0-alpha.2 rollback to @microsoft/teamsfx-cli: 0.3.0-alpha.0 
 - @microsoft/teamsfx-core: 0.4.0-alpha.2 rollback to - @microsoft/teamsfx-core: 0.3.0-alpha.0 
 - @microsoft/teamsfx: 0.4.0-alpha.3 rollback to @microsoft/teamsfx: 0.3.0-alpha.0 
 - ms-teams-vscode-extension: 2.3.0-alpha.3 rollback to ms-teams-vscode-extension: 2.2.0-alpha.0

### CODE REVIEW

**Review by LongOddCode — APPROVED** [review]

File: packages/sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/teamsfx",
-  "version": "0.4.0-alpha.3",
+  "version": "0.3.0-alpha.0",
```
Comment (tecton) [inline packages/sdk/package.json:3]: Do we need to rollback package-lock.json?

**Comment by LongOddCode** [comment]
Have we checked is there any legacy code in package-lock.json?

**Review by chagong — APPROVED** [review]

File: packages/sdk/package.json (hunk 1 above)
Comment (IvanJobs) [reply packages/sdk/package.json:3]: WIP

File: packages/sdk/package-lock.json [hunk 2]
```diff
@@ -7954,9 +8335,9 @@
       "dev": true
     },
     "trim-newlines": {
-      "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/trim-newlines/-/trim-newlines-3.0.1.tgz",
-      "integrity": "sha512-c1PTsA3tYrIsLGkJkzHF+w9F2EyxfXGo4UyJc4pFL++FMjnq0HJS69T3M7d//gKrFKwy429bouPescbjecU+Zw==",
+      "version": "3.0.0",
```
Comment (tecton) [inline packages/sdk/package-lock.json:8338]: This should be not be rollbacked? The old version has a security issue.

**Review by tecton — APPROVED** [review]

**Pull request merged**